.cargo/config.toml

```diff
@@ -21,6 +21,12 @@ IS_IN_RERUN_WORKSPACE = "yes"
 [target.wasm32-unknown-unknown]
 rustflags = ['--cfg=web_sys_unstable_apis', '--cfg=getrandom_backend="wasm_js"']
 
+[target.x86_64-apple-darwin]
+rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]
+
+[target.aarch64-apple-darwin]
+rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]
```

Comment:
what is this, what does it and why is it needed? ;)

Reply:
We definitely should not set _native_ RUSTFLAGS here, this will cause all sorts of nasty issues both for developers and CIs alike.